src/settings/watchers/FontWatcher.ts

```diff
@@ -24,30 +24,66 @@ import { UpdateSystemFontPayload } from "../../dispatcher/payloads/UpdateSystemF
 import { ActionPayload } from "../../dispatcher/payloads";
 
 export class FontWatcher implements IWatcher {
-    public static readonly MIN_SIZE = 8;
-    public static readonly DEFAULT_SIZE = 10;
-    public static readonly MAX_SIZE = 15;
-    // Externally we tell the user the font is size 15. Internally we use 10.
-    public static readonly SIZE_DIFF = 5;
+    /**
+     * Value indirectly defined by Compound.
+     * All `rem` calculations are made from a `16px` values in the
+     * @vector-im/compound-design-tokens package
+     *
+     * We might want to move to using `100%` instead so we can inherit the user
+     * preference set in the browser regarding font sizes.
+     */
+    public static readonly DEFAULT_SIZE = 16;
+    public static readonly MIN_SIZE = FontWatcher.DEFAULT_SIZE - 5;
+    public static readonly MAX_SIZE = FontWatcher.DEFAULT_SIZE + 5;
 
     private dispatcherRef: string | null;
 
     public constructor() {
         this.dispatcherRef = null;
+        this.migrateBaseFontSize();
```

Comment:
Think this should be in `start` or even later - this class is constructed at MatrixChat construction time, which is before we have loaded the initial sync, without loading that we don't have access to Account level settings. See note in Settings.tsx for more context re account level on the old setting